src/kontrol/options.py

```diff
@@ -381,7 +390,7 @@ def default() -> dict[str, Any]:
             'reinit': False,
             'bmc_depth': None,
             'run_constructor': False,
-            'use_gas': False,
+            'usegas': False,
             'break_on_cheatcodes': False,
             'recorded_diff_state_path': None,
```

Comment:
Since we're inverting the default here, can you double-check that the `--use-gas` option works locally, and that it by default is turned off?

Reply:
Will check, thanks!